Content.Shared/Fluids/SharedDrainSystem.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Content.Shared.DoAfter;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Fluids;
+
+public class SharedDrainSystem : EntitySystem
+{
+    [Serializable, NetSerializable]
+    public sealed class DrainDoAfterEvent : SimpleDoAfterEvent
```

Comment:
this is very minor so i will just merge, but in the future dont put events inside systems, just make them share namespace